translations/validateSyncSetup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+if [ ! -d "temp" ]; then
+	mkdir "temp"
+fi
+
+cd temp
+echo $PWD
+
+versions='main master stable31 stable30 stable29'
+if [ -f '.tx/backport' ]; then
+	versions="main master $(cat .tx/backport)"
+fi
+
+git clone git@github.com:$1/$2
+cd $2
+
+for version in $versions
+do
+	# skip if the branch doesn't exist
+	if git branch -r | egrep "^\W*origin/$version$" ; then
+		echo "Validating branch: $version"
+		echo "---"
+	else
+		echo "Invalid branch: $version"
+		continue
+	fi
+	git checkout $version
```

Comment:
maybe check out origin/version to validate what's on the server in case local branches haven't been updated?

Reply:
There is a git clone in line 15 :see_no_evil: